azure-pipelines/integrationtests.yml

```diff
@@ -1,56 +1,72 @@
 pool:
-    name: SqlToolsTestAgentPool
-    demands: VisualStudio_16.0
+    name: ads-build-1es-hosted-pool
+    demands: 
+    - ImageOverride -equals ADS-Linux_Image
 
 steps:
-  - task: DownloadSecureFile@1
-    displayName: 'Download secure file'
+
+  - task: DockerInstaller@0
+    displayName: Installing Docker
     inputs:
       secureFile: 'testsettings.json'
+      dockerVersion: 17.09.0-ce
+      releaseType: stable
 
-  - task: VisualStudioTestPlatformInstaller@1
-    displayName: 'Visual Studio Test Platform Installer'
-    inputs:
-      versionSelector: latestStable
+  - script: docker pull mcr.microsoft.com/mssql/server:2019-latest
+    displayName: Pull MSSQL Docker Image  
+
+  - bash: echo "##vso[task.setvariable variable=defaultSql2016_password;issecret=true]Test-$(Build.BuildNumber)-$(Get-Date -format yyyyMMdd-Hmmss)"

```

Comment:
Since this is for a 2019 server we should be setting this to `defaultSql2019_password`

To start with it's fine to disable the tests for older versions of SQL, we can work to enable targeting multiple versions later.